buildimage.py

```diff
@@ -148,16 +153,43 @@ def syncthing_init():
 
     # Check if syncthig is already cloned and pull latest if present else clone
     if os.path.isdir(syncthing_dir):
-        log.info("Pulling from syncthing:master...")
+        log.info("Pulling from syncthing repository...")
 
         cmd = "cd " + syncthing_dir + " && git pull"
         run_cmd(cmd)
     else:
-        log.info("Cloning syncthing:master...")
+        log.info("Cloning syncthing repository...")
 
-        cmd = "git clone -b v1.0.0 https://github.com/syncthing/syncthing " + syncthing_dir
+        cmd = "git clone --depth 1 -b v1.0.0  https://github.com/syncthing/syncthing " + syncthing_dir
         run_cmd(cmd)
 
+    cmd = "go get -u {}/...".format(syncthing_dir)
+    run_cmd(cmd)
+
+def do_build_plugin(plugin_name="ekstep", variant="diksha", prod=False):
+    repo_url = "https://github.com/projectOpenRAP/EkStep.git"
+    repo_name = "tmp"
+
+    plugins_root_dir = "{}/appServer/plugins".format(base_dir)
+    plugin_repo_dir = "{}/{}".format(plugins_root_dir, repo_name)
+    plugin_dir = "{}/{}".format(plugin_repo_dir, plugin_name)
+
+    build_type = "prod" if prod else "staging"
+
+    if (
+        plugin_name is not None and \
+        plugin_name in supported_plugins
+        ):
+
+        cmd = "rm -rf {} && git clone --depth 1 -b build-img-script {} {}".format(plugin_repo_dir, repo_url, plugin_repo_dir)
+        run_cmd(cmd)
+
+        cmd = "{}/init.sh {} {} {} {} {}".format(plugin_dir, plugin_name, variant, repo_name, base_dir, build_type)
```

Comment:
Make init.sh more meaning-full file name. Maybe call it as plugin_init.sh OR plugin_initilize.sh or plugin_copyconfigs.sh